introductory/src/App.jsx

```diff
@@ -1,8 +1,24 @@
-
+import Button from "./buttonComponent"
 
 function App() {
+
+
+    function handleClick(id){
+      alert(`You Just Clicked Button ${id}`);
+    }
+
+    const createComponents = (n) => {
```

Comment:
Try to be **consistency** in choosing which **type of function** to use between **regular** and **arrow** function